src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculator.kt

```diff
@@ -95,6 +98,7 @@ class ProtectLevelCalculator(
       .distinct()
       .filter { it != ComplexityFactor.IOM_NOMINAL }
       .count()
+      .times(2)
```

Comment:
Is this a new requirement?

Reply:
I've moved them from calling method into the callee as they're two separate requirements that we store the results of individually.

Reply:
👍